sequencer/src/persistence/fs.rs

```diff
@@ -408,6 +412,86 @@ impl SequencerPersistence for Persistence {
             },
         )
     }
+    async fn append_quorum_proposal(
+        &mut self,
+        proposal: &Proposal<SeqTypes, QuorumProposal<SeqTypes>>,
+    ) -> anyhow::Result<()> {
+        let view_number = proposal.data.view_number().u64();
+        let dir_path = self.proposals_dir_path();
+
+        fs::create_dir_all(dir_path.clone()).context("failed to create proposals dir")?;
+
+        let file_path = dir_path.join(view_number.to_string()).with_extension("txt");
```

Comment:
Do we ever prune old ones?

Reply:
If it is correct to delete anything equal or less than the decided view (I don't know if that's what we want, but if it is) it should be to just add this to the `delete_files` calls in `collect_garbage`. And likewise the `DELETE` statements in `collect_garbage` in `persistence/sql.rs`

Reply:
Yeah I think that's the case, I can update that.